.circleci/config.yml

```diff
@@ -0,0 +1,173 @@
+version: 2
+
+jobs:
+
+  test:
+    docker:
+      - image: rust:latest
+    steps:
+      - checkout
+      # Load cargo target from cache if possible.
+      # Multiple caches are used to increase the chance of a cache hit.
+      - restore_cache:
+          keys:
+            - v1-cargo-cache-test-{{ arch }}-{{ .Branch }}
+            - v1-cargo-cache-test-{{ arch }}
+
+      # Install nightly & wasm
+      - run:
+          name: Install Rust nightly
+          command: rustup update nightly && rustup default nightly
+      - run:
+          name: Add wasm32 target
+          command: rustup target add wasm32-unknown-unknown
+
+      # Install wasm tools
+      - run:
+          name: Install wasm-bindgen-cli
+          command: >
+            curl -OL https://github.com/rustwasm/wasm-bindgen/releases/download/0.2.33/wasm-bindgen-0.2.33-x86_64-unknown-linux-musl.tar.gz
+            && tar xf wasm-bindgen-0.2.33-x86_64-unknown-linux-musl.tar.gz
+            && chmod +x wasm-bindgen-0.2.33-x86_64-unknown-linux-musl/wasm-bindgen
+            && mv wasm-bindgen-0.2.33-x86_64-unknown-linux-musl/wasm-bindgen* $CARGO_HOME/bin/
+      - run:
+          name: Install wasm-pack
+          command: >
+            curl -L https://github.com/rustwasm/wasm-pack/releases/download/v0.6.0/wasm-pack-v0.6.0-x86_64-unknown-linux-musl.tar.gz
+            | tar --strip-components=1 --wildcards -xzf - "*/wasm-pack"
+            && chmod +x wasm-pack
+            && mv wasm-pack $CARGO_HOME/bin/
+
+      # Show versions
+      - run:
+          name: Show versions
+          command: rustc --version && cargo --version && wasm-bindgen --version && wasm-pack --version
+
+      # Install node and browser testing tools
```

Comment:
Same with `nodejs` - we no longer use it now that we use `--no-modules` for the example so this can be removed